datacite/jsonutils.py

```diff
@@ -18,14 +18,13 @@
 
 def validator_factory(schema_filename):
```

Comment:
Minor detail, but these changes are leaving all the import from ``os`` unused. They are not showed in this diff, but are the following:
``` python 
from os.path import abspath, basename, dirname, split
```
Maybe is a good idea to remove them to keep it clean.